tests/Functional/Command/SendTelemetryCommandTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Tests\Functional\Command;
+
+use Buddy\Repman\Command\SendTelemetryCommand;
+use Buddy\Repman\Tests\Functional\FunctionalTestCase;
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\Console\Tester\CommandTester;
+
+final class SendTelemetryCommandTest extends FunctionalTestCase
+{
+    public function testSendTelemetryWithoutInstanceIdFile(): void
+    {
+        @unlink($this->instanceIdFile());
+
+        $commandTester = new CommandTester(
+            $this->container()->get(SendTelemetryCommand::class)
+        );
+
+        self::assertEquals(0, $commandTester->execute([]));
+    }
+
+    public function testSendTelemetryWithTelemetryDisabled(): void
+    {
+        $this->generateInstanceIdFile();
+        $this->fixtures->changeConfig('telemetry', 'disabled');
+
+        $commandTester = new CommandTester(
+            $this->container()->get(SendTelemetryCommand::class)
+        );
+
+        self::assertEquals(0, $commandTester->execute([]));
```

Comment:
then maybe we would add to this `FakeTelemetryEndpoint` methods that would check if the send was called, because these tests now all have the same assertion ...

Reply:
So now you can add here:
```php
self::assertFalse($this->container()->get(TelemetryEndpoint::class)->sent());
```